typescript/src/entity.ts

```diff
@@ -90,6 +97,7 @@ interface ListOptions {
   clientId?: number;
   clientCompanyId?: number;
   savedByUser?: number;
+  serialiseMethod?: SerialiseMethod;
```

Comment:
are more than two "serialise methods" expected to ever exist? doesn't it make more sense to just use a boolean flag?

Reply:
yes I have considered there are might be more than one option happens in the future, so I think use a boolean is not as extenisble compare to use options